client/src/components/Picture/UploadPic.js

```diff
@@ -54,9 +71,12 @@ const UploadPic = ({ cameraIconSize, user }) => {
         image.onload = () => {
           if (image.height < 250 || image.width < 250) {
             setUploadError(
-              "Please upload an image that is atleast 250 pixels tall and 250 pixels wide.",
+              "Please upload an image that is at least 250 pixels tall and 250 pixels wide.",
+            );
+          } else if (image.height / image.width < 0.5) {
+            setUploadError(
+              "Please upload an image of aspect ratio 2:1 or better.",
```

Comment:
I wonder if the average user would know what this means 🤔 . Perhaps this could be rephrased as "The image is too long or too tall. Please upload an image where the height is less than twice the width or the width is less than twice the height."

Reply:
@mannykary Yes, I was thinking of using a different message for that. @skrithiga 

No, because if the height is more than twice the width, the overflow can be scrollable, and they can crop it that way.

Reply:
Got, in that case the copy can be simplified 

"The image is too wide. Please upload an image where the width of the image is less than twice the height"

Reply:
Yes, I still think this is going to be too complicated and technical from a user's perspective but I don't think practically may are going to use such images or come across the message.